internal/cmd/gocdk/launch.go

```diff
@@ -238,7 +220,7 @@ func (crl *cloudRunLauncher) Launch(ctx context.Context, input *LaunchInput) (*u
 	if err != nil {
 		return nil, xerrors.Errorf("cloud run launch: %w", err)
 	}
-	if err := crl.dockerPush(ctx, imageRef); err != nil {
+	if err := crl.dockerClient.Push(ctx, imageRef, ioutil.Discard); err != nil {
```

Comment:
Does `Push` need the arg then? IIUC this is the only place it's used.

Reply:
This function doesn't have direct access to the `processContext` and I didn't want to plumb through the `io.Writer` since I was intending to airlift this code to another package anyway. Adding a TODO for now.